### PR TITLE
Clarity for hivebot sizes.

### LINKED ISSUE #7630: Nice job on the Hivebot description
image

### PR DESCRIPTION
**Pull request by Probe1**
Closes https://github.com/d3athrow/vgstation13/issues/7630

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /mob/living/simple_animal/hostile/hivebot
 	name = "Hivebot"
-	desc = "A small robot"
+	desc = "A big robot"
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/hivebot.dm]: Period

**Review by Exxion — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm (hunk 1 above)
Comment (Probe1) [reply code/modules/mob/living/simple_animal/hostile/hivebot.dm]: ``` desc = "A big robot.  Period." ```

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm [hunk 2]
```diff
@@ -33,14 +33,14 @@
 
 /mob/living/simple_animal/hostile/hivebot/range
 	name = "Ranged Hivebot"
-	desc = "A smallish robot, this one has a gun!"
+	desc = "A biggish robot, this one has a gun!"
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/hivebot.dm]: COMMA SPLICE

Applies to the next one, too

Sorry for making this a separate review; I didn't notice this before.

**Review by Exxion — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm (hunk 2 above)
Comment (Probe1) [reply code/modules/mob/living/simple_animal/hostile/hivebot.dm]: 🔫

**Review by Exxion — APPROVED** [review]

**Pull request merged**